pages/resources/quests.mdx

```diff
@@ -365,19 +400,67 @@ Retrieves the reward code for the specified platform and returns a [quest reward
   Send Quest Heartbeat
 </RouteHeader>
 
-Tells the server to update the `last_stream_heartbeat` and `stream_progress_seconds` fields. Used for keeping track of how long the stream has been running for, and for checking if the user has met the [stream duration requirement](#quest-config-structure).
+Tells the server to update the `last_stream_heartbeat` and `stream_progress_seconds` fields. Used for keeping track of how long the stream has been running for, and for checking if the user has met the [stream duration requirement](#quest-config-structure). Returns a [quest user status](#quest-user-status-object) object.
 
 <Alert type="info">
 
 The `stream_progress_seconds` field is incremented by the amount of seconds since the last heartbeat request, up to 2 minutes at once.
 
 </Alert>
 
-Returns a [quest user status](#quest-user-status-object) object.
+<Alert type="warn">
+
+Quests may only be completed from desktop clients. If the requesting user-agent does not include an `Electron/` version, the request will fail with a 401 unauthorized error.
+
+</Alert>
 
 ###### JSON Params
 
-| Field      | Type    | Description                                                                                           |
-| ---------- | ------- | ----------------------------------------------------------------------------------------------------- |
-| stream_key | string  | The encoded key of the stream (e.g `guild:169256939211980800:1050497861969793164:222069018507345921`) |
-| terminal?  | boolean | Whether this is the last heartbeat in the current sequence                                            |
+| Field          | Type    | Description                                                                                           |
+| -------------- | ------- | ----------------------------------------------------------------------------------------------------- |
+| stream_key ^1^ | string  | The encoded key of the stream (e.g `guild:169256939211980800:1050497861969793164:222069018507345921`) |
+| terminal?      | boolean | Whether this is the last heartbeat in the current stream's sequence                                   |
+
+^1^ The stream must have at least 1 viewer to be eligible for quest progress.
```

Comment:
This is technically incorrect. The vc needs another participant, but they don't have to be viewing the stream.
Having a bot join the vc works for example.